google/resource_compute_instance_group_manager.go

```diff
@@ -121,10 +127,27 @@ func getNamedPorts(nps []interface{}) []*compute.NamedPort {
 			Port: int64(np["port"].(int)),
 		})
 	}
+
 	return namedPorts
 }
 
+func getNamedPortsBeta(nps []interface{}) []*computeBeta.NamedPort {
+	namedPorts := make([]*computeBeta.NamedPort, 0, len(nps))
+	for _, v := range nps {
+		np := v.(map[string]interface{})
+		namedPorts = append(namedPorts, &computeBeta.NamedPort{
+			Name: np["name"].(string),
+			Port: int64(np["port"].(int)),
+		})
+	}
+
+	return namedPorts
+}
+
+var InstanceGroupManagerBaseVersion = v1
```

Comment:
Can we bump this up to above the schema declaration? It's kind of tucked away here on line 147, all by itself. 

Also, but not required, I'd name it something that describes it's function/intent better, so specifically calling out that we're setting the base API version that the resource is talking to. Without getting too wordy, something like `igmBaseAPIVersion`. Not required though.